Index: src/components/Counter/Counter.tsx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import React, {useState} from 'react';\n\nimport classes from './Counter.module.css'\n\nexport const Counter = () => {\n\n    const [count, setCount] = useState<number>(0);\n    const increment = () => {\n        setCount(prev => prev + 1);\n    }\n\n    return (\n        <>\n            <h1 className={classes.count}>{count}</h1>\n            <button className={classes.incrementBtn} onClick={increment}>Increment</button>\n        </>\n    );\n};\n\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/components/Counter/Counter.tsx b/src/components/Counter/Counter.tsx
--- a/src/components/Counter/Counter.tsx	(revision 9a6422ac41e1f2c44b4eea7179543f01e3bc32fc)
+++ b/src/components/Counter/Counter.tsx	(date 1663176352801)
@@ -1,6 +1,6 @@
 import React, {useState} from 'react';
 
-import classes from './Counter.module.css'
+import classes from './Counter.module.scss'
 
 export const Counter = () => {
 
diff --git a/src/components/Counter/Counter.module.css b/src/components/Counter/Counter.modules.scss
rename from src/components/Counter/Counter.module.css
rename to src/components/Counter/Counter.modules.scss
